docs/docs/quickstart.md

```diff
@@ -8,8 +8,9 @@ sidebar_position: 2
 
 Before getting started with Eliza, ensure you have:
 
-- [Node.js 23.3.0](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-- [pnpm](https://pnpm.io/installation)
+- [Python 2.7+](https://www.python.org/downloads/)
```

Comment:
what requires python 2.7?

Reply:
I don't know - we shouldn't have this here if we don't need it.